src/components/AddItemForm.js

```diff
@@ -65,6 +65,8 @@ const Form = ({ token }) => {
                     tokenRef.doc(docRef.id).update({ id: docRef.id });
                     getCurrentShoppingListItems(userToken);
                     setAddStatus(true);
+                    alert(' Item Added!🙌');
+                    if(!alert('Item Added!🙌')){window.location.reload();}
```

Comment:
One tiny bug. When I add an item to the list, I end up seeing the `alert` message twice, and then I get the following on the Netlify site:  
![image](https://user-images.githubusercontent.com/10378331/81130011-d8a00780-8f0b-11ea-9999-910733b1beca.png)...I think because it's hitting line 69 and doing the `window.location.reload()`

Reply:
Weird.  I see the alert message twice, and it redirects me back to the add item form again.

Reply:
yes, slight bug with the form! we refreshed the window.location to reload, but the links seem broken when that happens. Will need to modify.

Reply:
Noticed Materialize has a 'Toast' feature, will play around with it to see if we can utilize this instead of the alert

Reply:
gonna update a separate issue for this actually